dist/rpm/openQA-test.spec

```diff
@@ -23,6 +23,11 @@ touch %{_sourcedir}/%{short_name}
 # call one of the components but not openqa itself which would need a valid
 # configuration
 /usr/share/openqa/script/initdb --help
+
+# verify whether assets can be loaded
+perl -I/usr/share/openqa/lib -mMojolicious -mMojo::Home -mOpenQA::Setup \
+    -e 'OpenQA::Setup::setup_asset_pack(Mojolicious->new(home => Mojo::Home->new("/usr/share/openqa")))'
```

Comment:
```suggestion
perl -I/usr/share/openqa/lib -mMojolicious -mMojo::Home -mOpenQA::Setup \
    -e 'OpenQA::Setup::setup_asset_pack(Mojolicious->new(home => Mojo::Home->new("/usr/share/openqa")))' || \
     { echo "assets could not be loaded"; exit $ERRCODE; }
```

Reply:
If this command fails it will already print a useful error message. So I wouldn't add another one.

Reply:
sure but I was trying to get rid of the comment with executable code :)